Add the colors to the tailwind configuration

avramkji opened the pull request:
Followed documentation as described in https://github.com/orgs/42dotmk/projects/6/views/1?pane=issue&itemId=60737413

avramkji committed:
Add the colors to the tailwind configuration

dimeskigj commented on astro/tailwind.config.js:
```diff
@@ -4,17 +4,35 @@ module.exports = {
       colors: {
+        black: '#000000',
+        white: '#FFFFFF',
+        blue: {
```
```suggestion
        ally-blue: {
```

avramkji committed:
tailwind colors config fix

- Old colors restored
- Theme colors (from figma) renamed to a11y as per request
- Theme colors used in Testimonial components at index.astro
